databases/week1/MySQl-Homework-Week1.sql

```diff
@@ -0,0 +1,169 @@
+	29	Sign up for linkedin	Make the CV awesome! 😄	2017-09-04 00:57:47	2017-10-18 18:07:48	2017-12-07 23:04:38	3	2
+	8	Walk the dog		2017-09-03 02:47:17	2017-10-12 18:40:08		3	2
+*/
+-- 6.Get the single most recently created task
+select * from task order by created desc limit 1;
+-- OUTPUT: 
+-- 	25	Look at apartments in Ørestad	2 or 3 rooms	2017-10-30 09:47:00	2017-10-19 06:11:26		1	6
+
+-- 7. Get the title and due date of all tasks where the title or description contains database
+select title, description, due_date from task where title like '%database%' || description like '%database%';
+-- OUTPUT: 
+/* 
+	title	description	due_date
+	Setup salary databases for accounting	Use MySQL	2017-12-05 00:19:08
+	Learn how databases work		2017-12-18 05:08:05
+	Make the databases perform better	It should be possible to optimize the indexes	2017-12-01 13:28:35
+	Backup databases to external disk	Remember to store the disk in another physical location	2017-12-23 14:21:01
+	Learn about NoSQL databases	MongoDB, CouchDB, etc.	2017-12-23 10:13:42
+*/
+-- 8. Get the title and status (as text) of all tasks
+select task.title,status.name as status from task join status on task.status_id=status.id;
+-- OUTPUT: 
+/*
+Order groceries online	Not started
+Do HackYourFuture homework	Not started
+Setup salary databases for accounting	Not started
+Charge electric bicycle	Not started
+Look at apartments in Ørestad	Not started
+Buy new dining room table and chairs	Not started
+Backup databases to external disk	Not started
+Buy plane ticket to Auckland	Not started
+Wash clothes	In progress
+Plan meeting with London office	In progress
+Empty the mailbox	In progress
+Fix the flat tire on the bike	In progress
+Wash the car	In progress
+Write a book	In progress
+Water the potted plants	In progress
+Buy wine for the birthday party	In progress
+Wash windows	In progress
+Make the databases perform better	In progress
+Knit sweater	In progress
+Ride bike aroud Sjælland	In progress
+Empty Mr Fluffy's litterbox	In progress
+Renew buscard	In progress
+Learn about NoSQL databases	In progress
+Become a billionaire	Done
+Walk the dog	Done
+Iron shirts	Done
+Buy gift for Paul	Done
+Change lightbulb in hallway	Done
+Learn how databases work	Done
+Buy beer for the company party	Done
+Buy new phone	Done
+Sign up for linkedin	Done
+Remove facebook from phone	Done
+Put up the new lamp in the hallway	Done
+Hang up paintings in living room	Done
+*/
+-- 9. Get the name of each status, along with a count of how many tasks have that status
+select status.name, count(task.id) as "status-count" from task join status on task.status_id=status.id group by status.id;
+-- OUTPUT: 
+/*
+	name	status-count
+	Not started	8
+	In progress	15
+	Done	12
+*/
+-- 10. Get the names of all statuses, sorted by the status with most tasks first
+select status.name, count(task.id) as "status-count" from task join status on task.status_id=status.id group by status.id order by task.id;
```

Comment:
here we don't get any data back, take a look at ORDER BY clause here (https://www.mysqltutorial.org/mysql-order-by/)